ep/lh/ep-lh-0009.md

```diff
@@ -0,0 +1,140 @@
+| -            | -           |
+| catalog      | Get engines catalog |
+| install      | Install engine (or multiple engines) |
+| help         | Show help   |
+| version      | Get lhman version |
+
+> By default, space is used as delimiter between argument and value. The equal sign is also admissible. If the argument is at the same time several values, then it is recommended to use the comma sign to separate them. In addition, multiple declaration of arguments is allowed as an alternative. Example: `lhman strategy --arg1=value1,value2 --arg2 value1 --arg2=value2`
+
+### catalog
+
+This strategy generates 
+
+| Argument    | Type     | Required | Description |
+| -           | :-:      | :-:      | -           |
+| filter      | string   | -        | Engine name (or engine id) to install [ep-lh-0002](#related) |
+| keys        | string   | -        | Engine version (latest version by default) |
+| bundle      | string   | -        | Generate a bundle with `metadata`, `arguments` and `dependencies` for all engines. Argument requires file name as string. |
+| engine      | string   | -        | Engine name to return `metadata`, `arguments` and `dependencies` |
+| project     | string   | -        | Project path to return list of recommended for installing engines |
+
+Here is command to get only `name` and `description` properties of each supported engine.
+
+`lhman catalog --filter=name,description`
+
+```json
+{
+    "linters": [
+        {
+            "name": "alex",
+            "description": "Catch insensitive, inconsiderate writing.",
+        },
+        {
+            "name": "bandit",
+            "description": "Security oriented static analyser for python code.",
+        },
+        ...
+    ]
+}
+```
+
+### install
+
+The install strategy allows to download engine and prepare necessary environment. 
+
+| Argument    | Type     | Required | Description |
+| -           | :-:      | :-:      | -           |
+| engine      | string   | +        | Engine name (or engine id) to install [ep-lh-0002](#related) |
+| version     | string   | -        | Engine version (latest version by default) |
+| project     | string   | -        | Project to associate with |
+| environment | string   | -        | The way how to install engine. Allowable values: `local`, `global`, `docker`. `local` is used by default. |
+
+There are 3 main ways how to deploy engine:
+* As global package. After installing the engine, it will be available globally in the entire system. Remember that this action may require root privileges. For this reason, this type of installation is not recommended.
+* As local package for current project. The engine will be installed in the location used by the `package manager` to store the project dependencies.
+* Using docker. This action will create a docker container in which the engine will be installed with all the necessary environment for work (based on [alpine](https://hub.docker.com/_/alpine/) docker image)
```

Comment:
`container` is probably a better name and `alpine` as a base container should be ideally configurable via some project config (or like that `--env=container --os=alpine`).